picard/cluster.py

```diff
@@ -269,6 +279,30 @@ def cluster(files, threshold):
 
             yield album_name, artist_name, (files[i] for i in album)
 
+    def update_metadata_images(self):
```

Comment:
This code is duplicated in https://github.com/metabrainz/picard/pull/657/files#diff-d6e5ee884fbd0a82f4c76dcd8584b6aaR227 
Not sure if this is stricly needed.